CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog - FeedHenry Javascript SDK
 
+## 3.0.1 - 2018-04-24
+* Travis upgrade NodeJS version configuration to 6.13.11.
```

Comment:
this should be 6.11.3?

Reply:
Yes. It should :-)

Reply:
Is a changelog entry really needed, given that this isn't something that needs a release? As I understand it, it doesn't actually change anything for the user, so I'm not sure what the difference to them between 3.0.0 and 3.0.1 would be.

Reply:
@grdryn good point. I agree with you :-).

Reply:
@grdryn regards the point 1. 

The problem is we are using a deprecated version to run the test and build the app. We should follow the version adopted which is 6.13.11 for many reasons, following some points:

* To fix something, for example, it can not work with the version 0.10. 
* We will test locally with the 6.13.11 by default and may we will be not able to merge the PR ( version incompatibilities )
* Also, the npm manager used to build, test and etc .. have been improved and has fixed bug in the newer versions, as better outputs,  and etc ..
* if we need to solve a CEV the NodeJS used here can affect it and don't' allow we do the merge of it
* it is used for the client application and the users will be not using the version 0.10 deprecated to them and it may cause some impacts too

Reply:
It really doesn't matter which version of Node since it is only to run our own tests etc. It should be a supported version of Node at least. Weather that is 6 or 8, I have no preference on.

Reply:
Agreed with Ger on a 3.0.1. I would like to have all changes listed in the CHANGELOG but for this, please add it to the Unreleased section as specified in https://keepachangelog.com/en/1.0.0/ and it can be added to 3.0.1 once there is a change that requires a release to npm.